.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Collapse changes to Polly recordings in PRs.
+tests/__recordings__/** linguist-generated
```

Comment:
... huh. Part of our typical repo setup s a `.gitattributes` file that sets `* text=auto`, to normalize line endings. We also instruct linguist not to collapse lockfile changes.

I guess we forgot it here.